docs/variants/dell_optiplex/initial-deployment.md

```diff
@@ -163,22 +145,22 @@ after adding blobs
 
 ## Flashing
 
-Following procedure will flash Dell OptiPlex 7010/9010 SFF Dasharo firmware to your
-machine SPI NOR flash.
+Following procedure will flash Dell OptiPlex 7010/9010 SFF Dasharo firmware to
+your machine SPI NOR flash.
```

Comment:
This content is shared with Arch4032, so I want to be very precise especially that I give lecture about what is typically used for firmware storage. I know this may seem as over precision, but maybe we can leave it?